test/client_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "./test_helper"
+require_relative "../src/clients/kdump"
+ 
+describe Yast::KdumpClient do
+ 
+  describe ".cmdKdumpStartup" do
+    let(:alloc_mem_high) {"100"}
+    let(:alloc_mem_low) {"50"}
+    
+    before(:each) do
+      subject.cmdKdumpStartup(options)
+    end
+   
+    context "When using alloc memory high and low" do
+      let(:options) {{"enable"=>"", "alloc_mem"=>"#{alloc_mem_low},#{alloc_mem_high}"}}
+    
+      it "sets the low and high values of  Kdump allocated_memory " do
+        expect(Yast::Kdump.allocated_memory).to eq({:low=>alloc_mem_low, 
+          :high=>alloc_mem_high})
+        expect(Yast::Kdump.add_crashkernel_param).to be true
+      end
+    end
+    
+    context "when using only alloc memory low" do
+      let(:options) {{"enable"=>"", "alloc_mem"=>"#{alloc_mem_low}"}}
+      
+      it "enables only the low value of Kdump allocated_memory " do
+        expect(Yast::Kdump.allocated_memory).to eq({:low=>alloc_mem_low, 
+          :high=>nil})
+        expect(Yast::Kdump.add_crashkernel_param).to be true
+      end
+    end
+    
+    context "When receive option disable" do
```

Comment:
I would make clear that 'disable' is an option: *When receives option 'disable'* or *when receives the 'disable' option* or...